NEWS

```diff
@@ -18,6 +18,20 @@ CHANGES WITH 253 in spe:
           For more details, see:
           https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
 
+        * We intend to change behaviour w.r.t. user units and sandboxing
+          options, so that they work in the same way as system units by default.
+          This requires implicitly enabling user namespaces (PrivateUsers=yes)
```

Comment:
"requires" suggests people have to do things here.

instead say "To make this work, we'll now implicitly enable" or so